prashant/HTML/Assignment5/AboutMe.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Class Attribute</title>
+        <link rel="stylesheet" href="aboutme.css">
+    </head>
+    <body>
+        <h1>Prashant Sahu</h1>
+        <hr>
+        <p class="p-about">About Me<p>
+        <div class="div-self-description">    
+            <p>My native place is Jabalpur MadhyaPradesh.</p>
+            <p>I have completed BE from Global Engineering Collage and Schooling from ST Mary's H.S. School.</p>
+            <p> we are 6 members in my family. My father is a Business man, my mother is a house wife, my 2 elder sister is maried and my younger sister was doing Bcom</p>
+        </div> 
+        <p class="p-more-about">MoreAbout Me</p>
+        <ul class="div-self-description">
+            <li>My Resume</li><br>
+            <li>Hobbies</li><br>
+            <ul>
```

Comment:
indentation missing

Reply:
done sir

Reply:
not done you need to give space before ul tag li is parent tag there and ul is child tag

Reply:
ok sir now i have done it please cheack